deployment/main.tf

```
@@ -268,14 +267,21 @@ resource "null_resource" "up" {
     destination = "${local.work_dir}/defaults.env"
   }
 
+  provisioner "file" {
+    source      = "${path.root}/systemd/system"
+    destination = "/etc/systemd/system"
+  }
+
```

j'ai déployé en staging, ça va dans `/etc/systemd/system/system/`

```suggestion
  provisioner "file" {
    source      = "${path.root}/systemd/system"
    destination = "/etc/systemd"
  }

```